diff_cover/violations_reporter.py

```diff
@@ -116,14 +132,14 @@ def _get_classes(self, xml_document, src_path):
              src_abs_path in [
                  os.path.join(
                      source,
-                     clazz.get('filename')
+                     self._to_unix_path(clazz.get('filename'))
```

Comment:
This bit is slightly off. `os.path.join` will join the two halves with the native separator, plus I'm not sure what the format of `source` is. Probably best to do the join first and call `self._to_unix_path` on the result of that.